src/Server/Provision/ServerProvisioner.php

```diff
@@ -163,26 +163,35 @@ public function check()
      */
     private function checkEntities()
     {
-        if ($entities = $this->getEntities()) {
-            return $entities;
+        $isCheck = true;
+        if ($entities = $this->getEntities($isCheck)) {
```

Comment:
This could return an empty array. We should instead throw an exception in the case that `count($entities) === 0`